neon/pupil-cloud/offset-correction/index.md

```diff
@@ -0,0 +1,19 @@
+# Offset Correction on Pupil Cloud
+
+For some subjects, you may find a constant offset in their gaze estimates. This gaze offset can be [compensated for in the Neon Companion app](https://docs.pupil-labs.com/neon/data-collection/offset-correction/) before starting a recording, or post hoc in Pupil Cloud as shown below. Click on the gaze offset icon to start the gaze offset correction.
+
+![Offset correction on Cloud header image](./offset-cloud-timeline.png)
```

Comment:
The icon is very small in the image and it's not entirely obvious where it is. Should we highlight it somehow? Could also be delegated to the design team.

Reply:
I added a new image with zoom-in, such that the icon looks a bit better/bigger. Would this work? Otherwise, we can also simply mention it in-text "Click on the gaze offset icon (next to the gaze stream as shown below)"

Reply:
I like this image better because the small things appear less pixelated, but it still doesn't quite make clear which button is the gaze offset button. Adding a description in text might be helpful, but a visual indicator (like a red arrow or something) would also be helpful. Do you see a reason to avoid something like that?

Reply:
![offset-cloud-timeline](https://github.com/pupil-labs/pupil-docs/assets/7340349/e459a654-c928-4137-b889-5e7945eb75f3)
Something like that? Let me know if you prefer assigning this to design.

Reply:
This looks good to me, let's just take this!